docs/aws-professional/services.md

```diff
@@ -22,168 +22,168 @@ For an overview of Azure for AWS users, see [Introduction to Azure for AWS exper
-| **&nbsp;**         | AWS IoT                                                                                 | [Azure IoT Hub](https://azure.microsoft.com/services/iot-hub/)                                                                             | A cloud gateway for managing bidirectional communication with billions of IoT devices, securely and at scale.                                                     |
-| **&nbsp;**         | AWS Greengrass - Software for Connected Devices                                         | [Azure IoT Gateway SDK](https://azure.microsoft.com/services/iot-hub/iot-gateway-sdk/)                                                     | Contains the infrastructure and modules to create IoT gateway solutions. The SDK can be used and extended to create gateways tailored to any end-to-end scenario. |
-| Streaming data     | Kinesis Firehose <br/><br/>Kinesis Streams                                                        | [Event Hubs](https://azure.microsoft.com/services/event-hubs/)                                                                             | Services that allow the mass ingestion of small data inputs, typically from devices and sensors, to process and route the data.                                   |
+| Internet of Things | AWS IoT Other Services (Kinesis, Machine Learning, EMR, Data Pipeline, SNS, QuickSight) | [IoT Suite (IoT Hub, Machine Learning, Stream Analytics, Notification Hubs, PowerBI)](https://azure.microsoft.com/suites/iot-suite/) | Provides a preconfigured solution for monitoring, maintaining, and deploying common IoT scenarios.                                                                |
+| **&nbsp;**         | AWS IoT                                                                                 | [IoT Hub](https://azure.microsoft.com/services/iot-hub/)                                                                             | A cloud gateway for managing bidirectional communication with billions of IoT devices, securely and at scale.                                                     |
+| **&nbsp;**         | AWS Greengrass - Software for Connected Devices                                         | [IoT Edge](https://azure.microsoft.com/services/iot-hub/iot-gateway-sdk/)                                                     | Contains the infrastructure and modules to create IoT gateway solutions.  |
+| Streaming data     | Kinesis Firehose <br/><br/>Kinesis Streams                                                        | [Event Hubs](https://azure.microsoft.com/services/event-hubs/)                                                                             | Allow mass ingestion of small data inputs, typically from devices and sensors, to process and route data.                                 |
 
 
-## Management & monitoring
+## Management & Monitoring
```

Comment:
original is correct. Microsoft docs style for heading is sentence casing

Reply:
OK